src/main/scala/tilelink/SRAM.scala

```diff
@@ -53,7 +56,22 @@ class TLRAM(
     val width = code.width(eccBytes*8)
     val lanes = beatBytes/eccBytes
     val addrBits = (mask zip edge.addr_hi(in.a.bits).asBools).filter(_._1).map(_._2)
-    val (mem, omMem) = makeSinglePortedByteWriteSeqMem(1 << addrBits.size, lanes, width)
+    val (mem, omSRAM, omMem) = makeSinglePortedByteWriteSeqMem(
+      size = 1 << addrBits.size,
+      lanes = lanes,
+      bits = width)
+
+    parentLogicalTreeNode.map {
+      case parentLTN =>
+        def sramLogicalTreeNode = new BusMemoryLogicalTreeNode(
+          device = device,
+          omSRAMs = Seq(omSRAM),
+          busProtocol = new TL_C(None),
+          dataECC = None,
+          hasAtomics = None,
```

Comment:
Why don't you set these to the actual values from the passed- in parameters? As written this will not produce correct OM information!

Reply:
fixed

Reply:
fixed

Reply:
I'm not seeing this as fixed, did you forget to push something?